### DISCUSSION TITLE
Volume normalization

### DISCUSSION THREAD

**Discussion by heyDaino** [enzo1982/freac#435] [Help & Support]
Hi everyone. Does Fre:ac have the ability to boost the volume? I need to convert a series of wave files normalized to -3dB and bring them down to MP3 -1dB
Cheers!

**Comment by enzo1982** [comment]
No actual normalization yet, but recent [continuous builds](https://github.com/enzo1982/freac/releases/continuous) have a _Volume Adjustment_ filter in the _Processing_ settings.

You can use that to increase or decrease volume by a static dB value.

**Comment by AlanHK** [comment]
Try wavegain to process the wave files first
https://github.com/MestreLion/wavegain

**Comment by mesheets** [comment]
Cross-references to normalization posts on the prior SourceForge site:
- https://sourceforge.net/p/bonkenc/discussion/390598/thread/d9f528eb/
- https://sourceforge.net/p/bonkenc/feature-requests/18/
